<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLineEdit" name="lName">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pbOpen">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string comment="File" extracomment="open File (GCode)">File</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QTextEdit" name="tbCode">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>SizeVerCursor</cursorShape>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="lineWrapMode">
         <enum>QTextEdit::NoWrap</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>669</width>
     <height>42</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumal">
    <property name="title">
     <string comment="what">what</string>
    </property>
    <addaction name="actionZeit"/>
    <addaction name="actionBild"/>
    <addaction name="actionWand"/>
   </widget>
   <widget class="QMenu" name="menuhier">
    <property name="title">
     <string>here</string>
    </property>
    <addaction name="actionnicht"/>
    <addaction name="actionoder_doch"/>
   </widget>
   <addaction name="menumal"/>
   <addaction name="menuhier"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionZeit">
   <property name="text">
    <string>ever</string>
   </property>
  </action>
  <action name="actionBild">
   <property name="text">
    <string>Picture</string>
   </property>
  </action>
  <action name="actionWand">
   <property name="text">
    <string>Wall</string>
   </property>
  </action>
  <action name="actionnicht">
   <property name="text">
    <string>not</string>
   </property>
  </action>
  <action name="actionoder_doch">
   <property name="text">
    <string>or after all?</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
